### PR TITLE
Add compile flags and deps to support the text drawing 

### PR DESCRIPTION
**Pull request by poetaster**
facilities in ffmpeg. Subtitles, text animation, and so on.

It seems all required deps. are available. 

This would allow text overlays and subtitles in Videoworks, for instance.

### CODE REVIEW

File: rpm/ffmpeg.spec [hunk 1]
```diff
@@ -11,6 +11,8 @@ License:        LGPLv2+
 BuildRequires:  pkgconfig(libopenjp2)
 BuildRequires:  pkgconfig(libpulse)
 BuildRequires:  pkgconfig(libwebp)
+BuildRequires:  pkgconfig(harfbuzz)
+BuildRequires:  pkgconfig(freetype)
```
Comment (mlehtima) [inline rpm/ffmpeg.spec:15]: This appears to be wrong, the pkgconfig name is freetype2 as seen here https://github.com/sailfishos/freetype/blob/master/rpm/freetype.spec#L72

Comment (mlehtima) [reply rpm/ffmpeg.spec:15]: Also could you keep the alphabetical ordering of the dependencies.

Comment (poetaster) [reply rpm/ffmpeg.spec:15]: > Also could you keep the alphabetical ordering of the dependencies.

Sure thing. I'll do a trial build on obs before resubmitting. Thanks!

Comment (poetaster) [reply rpm/ffmpeg.spec:15]: Oh, looking at: Provides:   %{name}-bytecode adds to the confusion (for me). I had used zypper to check if the packages were available and zypper responded with freetype and freetype-devel. 

I guess I'll go dig a bit more. Thanks again!

Comment (mlehtima) [reply rpm/ffmpeg.spec]: The harfbuzz option was added only in ffmpeg 6.1 and needs to be removed from this PR, probably the build requirement also for it. Maybe you had been testing with newer ffmpeg version than what we have here now.

**Pull request merged**